common/events.d.ts

```diff
@@ -6,8 +6,17 @@ interface SectionStartedEvent {
     section: SectionName;
 }
 
+interface MoveCameraEvent {
+    cameraTargetX?: number;
+    cameraTargetY?: number;
+    cameraTargetZ?: number;
+    unitTargetEntIndex?: EntityIndex;
+    lerp: number;
```

Comment:
What is lerp? Duration?

Reply:
Should probably be renamed "duration" and used as a duration - the time it takes for the camera to pan from the unit to the target.